stack/database.py

```diff
@@ -73,11 +73,12 @@
     db_name,
     ":",
     pulumi.Config("db").require_secret("db-password"),
-    "@/",
+    "@",
```

Comment:
I'd prefer to increase the memory limit for the container instances, because I think the intermittently occurring errors connecting to the DB might be related to an out-of-memory condition and not by using an invalid connection string. 

That the containers in the CloudRun Services run into out-of-memory conditions is to me a problem that has to be tackled anyways.